handbook/engineering/security/index.md

```diff
@@ -98,6 +98,24 @@ When we receive [a report of a security vulnerability](#how-to-report-a-security
 
 --------------------------
 
+## How we pay out bounties
+
+When a reported vulnerability is fixed, the security team will decide upon an appropriate bounty, and follow up with the reporter. A simple template response is
+
+> We have fixed the vulnerability you reported in the following PR: $PRLINK
+>This fix is included in the following versions of Sourcegraph: $SGVERSION
+
+>A bounty of $BOUNTYVAL will be awarded. To pay the bounty, we require the following information:
```

Comment:
nit: Just to make it crystal clear within the message since a recent reporter asked about it.

```suggestion
>A bounty of $BOUNTYVAL will be awarded. To pay the bounty through PayPal, we require the following information:
```